app/docker/views/containers/console/containerconsole.html

```diff
@@ -27,6 +27,7 @@
                       <option value="sh" ng-if="imageOS == 'linux'">/bin/sh</option>
                       <option value="powershell" ng-if="imageOS == 'windows'">powershell</option>
                       <option value="cmd.exe" ng-if="imageOS == 'windows'">cmd.exe</option>
+                      <option ng-repeat="command in containerCommands" value="{{command.command}}">{{command.title}}: {{command.command}}</option>
```

Comment:
What about simply displaying the command that will be used instead of showing the label title in there? Is this of any interest to you?

Reply:
titles could be handy stating what command is doing
`clear-cache`, `show-config`, `external-service-connectivity-check`
altho most of them can be guessed by reading the command but having title helps ops as some commands might not be easy to read/guess.
shall I replace `-_` with space in titles there?
my example above was using `_` but docker prefers `-`

Reply:
All good then